CHANGELOG.md

```diff
@@ -87,5 +89,6 @@ Initial Release
 [1.0.1]: https://github.com/Esri/hub.js/compare/v1.0.0...v1.0.1 "v1.0.1"
 [1.1.0]: https://github.com/Esri/hub.js/compare/v1.0.1...v1.1.0 "v1.1.0"
 [1.1.1]: https://github.com/Esri/hub.js/compare/v1.1.0...v1.1.1 "v1.1.1"
-[1.2.0]: https://github.com/Esri/hub.js/compare/v1.1.1...1.2.0 "1.2.0"
+[1.2.0]: https://github.com/Esri/hub.js/compare/v1.1.1...v1.2.0 "v1.2.0"
+[1.2.0]: https://github.com/Esri/hub.js/compare/v1.2.0...1.2.0 "1.2.0"
```

Comment:
here too.

my instinct is to bump the internal dependencies only in this PR and save the rest for the actual release script.